tests/lib/linter.js

```diff
@@ -71,7 +71,7 @@ function getVariable(scope, name) {
 // Tests
 //------------------------------------------------------------------------------
 
-describe("eslint", () => {
+describe("linter", () => {
```

Comment:
Should this be Linter instead? (Are we specifically testing the singleton instance, or the class?)